### PR TITLE
Released sdk

### PR DESCRIPTION
**Pull request by mootz12**

### CODE REVIEW

**Review by markuspluna — DISMISSED** [review]
lgtm

File: src/scripts/submit.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { Network, PoolClient, Request, TxOptions } from '@blend-capital/blend-sdk';
+import { config } from '../utils/env_config.js';
+import { AddressBook } from '../utils/address_book.js';
+import { logInvocation, signWithKeypair } from '../utils/tx.js';
+
+async function mock(addressBook: AddressBook) {
```
Comment (Ryang-21) [inline src/scripts/submit.ts:6]: Is this for debugging?

**Review by Ryang-21 — COMMENTED** [review]
Just had a question on the submit.ts file but otherwise looks good

**Review by Ryang-21 — APPROVED** [review]

File: src/scripts/submit.ts (hunk 1 above)
Comment (mootz12) [reply src/scripts/submit.ts:6]: I've found some use of having a file that just executes a function on the pool so I left it, but yes

**Pull request merged**